src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -764,15 +770,18 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
             ? virtualRuneFiatLiquidityAmount
             : virtualAssetFiatLiquidityAmount
 
+          const accountId = accountIdsByChainId[asset.chainId]
+
           return (
             <TradeAssetInput
-              key={_asset.assetId}
-              assetId={_asset?.assetId}
-              assetIcon={_asset?.icon ?? ''}
-              assetSymbol={_asset?.symbol ?? ''}
+              accountId={accountId}
```

Comment:
The only change of this diff - all the rest was making naming sane so we could avoid the `_asset` above, which we always seem to get wrong